packages/video_player/video_player_avfoundation/example/ios/RunnerTests/VideoPlayerTests.m

```diff
@@ -175,4 +225,47 @@ - (void)testHLSControls {
   return initializationEvent;
 }
 
+- (void)testTransformFixByOrientation:(UIImageOrientation)orientation {
```

Comment:
A helper method for a test shouldn't start with `test`, since that's the naming convention for actual test methods. How about `validateTranformFixForOrientation:`